pyro/distributions/relaxed_straight_through.py

```diff
@@ -29,10 +29,6 @@ class RelaxedOneHotCategoricalStraightThrough(RelaxedOneHotCategorical):
     [2] Categorical Reparameterization with Gumbel-Softmax,
         Eric Jang, Shixiang Gu, Ben Poole
     """
-    def __init__(self, temperature, probs=None, logits=None, validate_args=None):
```

Comment:
what's going on here? where did the temperatures go?

Reply:
Since its just calling its parent with the same arguments, its not needed. We can just rely on the parent's init method.